### PR TITLE
Tableau.Reader version 2021.2.1

### PR DESCRIPTION
**Pull request by walderston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22141)

### CODE REVIEW

File: manifests/t/Tableau/Reader/2021.2.1/Tableau.Reader.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: Tableau.Reader
+PackageVersion: 2021.2.1
+PackageLocale: en-US
+Publisher: Tableau
+PackageName: Tableau Reader
```
Comment (OfficialEsco) [inline manifests/t/Tableau/Reader/2021.2.1/Tableau.Reader.yaml:8]:
```suggestion
PackageVersion: 21.2.1241
PackageLocale: en-US
Publisher: Tableau Software
PackageName: Tableau Reader
```

File: manifests/t/Tableau/Reader/2021.2.1/Tableau.Reader.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ShortDescription: Tableau Reader is a free desktop application that you can use to open and interact with data visualisations built in Tableau Desktop
+Moniker: tableau-reader
+Tags:
+- charts
+- graphs
+- business
+- tableau
+Installers:
+- Architecture: x64
+  InstallerType: inno
```
Comment (OfficialEsco) [inline manifests/t/Tableau/Reader/2021.2.1/Tableau.Reader.yaml:21]:
```suggestion
  InstallerType: burn
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/t/Tableau/Reader/2021.2.1/Tableau.Reader.yaml (hunk 2 above)
Comment (walderston) [reply manifests/t/Tableau/Reader/2021.2.1/Tableau.Reader.yaml:21]: leaving this as InstallerType inno.
It successfully installs in the background without any popup (apart from the UAC)
With InstallerType burn, it displays the installation windows which is undesired

**Comment by OfficialEsco** [comment]
`Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\t\Tableau\Reader\2021.2.1', expected 'manifests\t\Tableau\Reader\21.2.1241'`

File: manifests/t/Tableau/Reader/2021.2.1/Tableau.Reader.yaml (hunk 2 above)
Comment (OfficialEsco) [reply manifests/t/Tableau/Reader/2021.2.1/Tableau.Reader.yaml:21]: huh, is this specific to this installer or does /SILENT and /VERYSILENT work for Burn?

**Pull request closed without merging**